service/src/main/java/bio/terra/workspace/service/job/JobService.java

```diff
@@ -134,14 +134,6 @@ protected <T> T submitAndWait(
     return resultOrException.getResult();
   }
 
-  protected <T> T submitAndWait(
```

Comment:
Unused overload, everything already calls the `submitAndWait(..., boolean doAccessCheck)` version instead